ansible/roles/couchdb/tasks/deploy.yml

```diff
@@ -5,6 +5,21 @@
   fail: msg="The db provider in your {{ inventory_dir }}/group_vars/all is {{ db_provider }}, it has to be CouchDB, pls double check"
   when: db_provider != "CouchDB"
 
+- name: check for persistent disk
+  shell: df -h
+  register: disk_status
+
+- name: (re)start CouchDB
+  docker:
+    name: couchdb
+    image: couchdb
+    state: reloaded
+    restart_policy: "{{ docker.restart.policy }}"
+    volumes: "/var/couchdb:/usr/local/var/lib/couchdb"
```

Comment:
As I read this, only this line is different from the "normal" "(re)start CouchDB". Can this be solved in a more DRY way?